### DISCUSSION TITLE
Customize Toolbar

### DISCUSSION THREAD

**Discussion by sheldorC137** [guoyingtao/Mantis#295] [Q&A]
Hello,
I wonder how I could change the toolbar appearance. 
1. Where can I change the reset Button Image? I searched for arrow.triangle.2.circlepath 
![photo](https://user-images.githubusercontent.com/118553188/229191645-41f8b851-38ed-4d5a-9f0e-3e9d688ed7b2.png)

but I didnt find that
2. Can I add a white background behind the done and cancel button? I didnt find the code position of these buttons

Thanks for help. Im not familiar with UIKit really well.

**Comment by guoyingtao** [comment]
@sheldorC137 
For question 1, you can set iconProvider of CropToolbar
I don't quite understand question 2. Do you only want to add white background for done and cancel button?

**Reply by sheldorC137** [reply]
Thanks. Yes only white background or a capsule for example
